.github/workflows/ci.yml

```
@@ -0,0 +1,30 @@
+on: 
+    pull_request:
+      branches: [ master ]
+    push:
+      branches: [ master ]
+name: Test
```

This forms part of the action name, would this work?
```suggestion
name: ci/github
```

---

updated in current push